tests/test_browser.py

```diff
@@ -3352,39 +3352,34 @@ def test_minimal_runtime_modularize(self):
 
   @requires_sync_compilation
   def test_modularize(self):
-    for opts in [[], ['-O1'], ['-O2', '-profiling'], ['-O2'], ['-O2', '--closure', '1']]:
+    for opts in [
+      [], 
+      ['-O1'], 
+      ['-O2', '-profiling'], 
+      ['-O2'], 
+      ['-O2', '--closure', '1']
+    ]:
       for args, code in [
-        ([], 'Module();'), # defaults
-        # use EXPORT_NAME
-        (['-s', 'EXPORT_NAME="HelloWorld"'], '''
-          if (typeof Module !== "undefined") throw "what?!"; // do not pollute the global scope, we are modularized!
-          HelloWorld.noInitialRun = true; // errorneous module capture will load this and cause timeout
-          HelloWorld();
+        # defaults
+        ([], '''
+          let promise = Module();
+          if (!promise instanceof Promise) throw new Error('Return value should be a promise');
         '''),
+        # use EXPORT_NAME
```

Comment:
This should still pass, but it's not I can't figure out for the life of me why not. I can't even figure out why it even passes at `HEAD`.

Reply:
Accidentally removed a line I shouldn't have, passes now